engine/configurator/conf_cyaml.c

```diff
@@ -0,0 +1,796 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "conf_cyaml.h"
+
+/******************************************************************************
+ * CYAML schema to tell libcyaml about both expected YAML and data structure.
+ *
+ * (Our CYAML schema is just a bunch of static const data.)
+ ******************************************************************************/
+
+/* Mapping from instance or object strings to enum values for schema. */
+const cyaml_strval_t scope_strings[] = {
```

Comment:
All these objects should be `static` or have a name prefix.

Reply:
Prefix was added.